File: src/main/java/net/softbell/bsh/dto/rest/ResultDTO.java
package net.softbell.bsh.dto.rest;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : API 수행 결과 DTO
 */
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ResultDTO
{
	private String message;
}

File: src/main/kotlin/net/softbell/bsh/dto/response/ResultDto.kt
package net.softbell.bsh.dto.response

import io.swagger.annotations.ApiModelProperty

/**
 * @author : Bell(bell@softbell.net)
 * @description : API 수행 결과 DTO
 */
open class ResultDto(
        @ApiModelProperty(value = "응답 성공여부")
        var success: Boolean = false,

        @ApiModelProperty(value = "응답 코드")
        var code: Int = 0,

        @ApiModelProperty(value = "응답 메시지")
        var message: String? = null
) {
}